x-pack/plugins/snapshot_restore/public/application/components/policy_form/steps/step_logistics.tsx

```diff
@@ -273,6 +273,7 @@ export const PolicyStepLogistics: React.FunctionComponent<StepProps> = ({
         }}
         fullWidth
         data-test-subj="repositorySelect"
+        disabled={policy?.isManagedPolicy && isEditing}
```

Comment:
[nit] We might even consider adding a [tooltip](https://eui.elastic.co/#/display/tooltip) when it's disabled with a brief explanation why

Reply:
Thanks, I will add the tooltips via https://github.com/elastic/kibana/issues/173124